cluster_test.go

```diff
@@ -1682,7 +1682,7 @@ func (s *S) TestMaxSocketUses(c *C) {
 	if *fast {
 		c.Skip("-fast")
 	}
-	session, err := mgo.Dial("localhost:40011?maxSocketUses=4")
```

Comment:
I wanted to add test case for my feature and took existing test as sample but didn't really create copy of test but instead modified the original :(. Will fix.